gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=11.1.0
+version=12.0.0
```

Comment:
Since I had to remove `errorRate` field from `ProviderAvailability` this is a breaking change